rust/src/ssh/ssh.rs

```diff
@@ -29,6 +30,14 @@ fn hassh_is_enabled() -> bool {
     HASSH_ENABLED.load(Ordering::Relaxed)
 }
 
+// app-layer-frame-documentation tag start: FrameType enum
+#[derive(AppLayerFrameType)]
+pub enum SshFrameType {
+    RecordHdr,
```

Comment:
Q: No other frames seem to have this naming scheme with `Record` prefix. Why did you choose it?

Reply:
Because SSH has different stages : banner, records, and encrypted

So, the frames are about records only

Reply:
oh. thank you!